Formula/ec2instanceconnectcli.rb

```diff
@@ -0,0 +1,69 @@
+class Ec2instanceconnectcli < Formula
+  include Language::Python::Virtualenv
+
+  desc "AWS EC2 Instance Connect CLI"
+  homepage "https://aws.amazon.com/cli/"
```

Comment:
On the homepage, ec2instanceconnectcli is not mentioned anywhere…

Reply:
Changed to https://github.com/aws/aws-ec2-instance-connect-cli